### PR TITLE
feat: add migration notes for 1.9.0

### PR DESCRIPTION
**Pull request by gpoujol**

### CODE REVIEW

File: docs/admin/upgrading_to_1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Upgrading to 1.9.0
+
+This note describes changes made by upgrade to Stellio 1.9.0
+
+## New endpoint for rights management.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:5]:
```suggestion
## New endpoints for rights management
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+Stellio 1.9.0 brings new endpoints to visualize the rights that we have on entities but also to know the groups which we belongs to.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:7]:
```suggestion
Stellio 1.9.0 brings new endpoints to visualize the rights an user has on entities but also to know the groups which he belongs to.
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+### Get authorized entites for the currently authenticated user 
+
+The first endpoint allows you to retrieve the rights for all the entities of the authenticated person.
+
+To do this, you must do a GET on `/ngsi-ld/v1/entityAccessControl/entities`.
+
+The following request parameters are supported: 
+
+    -  q: to restrict returned entities to the ones with a specific right. Only "rCanRead" and "rCanWrite" and "rCanAdmin" are accepted. A list is accepted (e.g, “q=rCanRead;rCanWrite”). This request parameter does not work when user are stellio admin.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:17]:
```suggestion
    -  `q`: to restrict returned entities to the ones with a specific right. Only "rCanRead" and "rCanWrite" and "rCanAdmin" are accepted. A list is accepted (e.g, “q=rCanRead;rCanWrite”). This request parameter does not work when user is Stellio admin. 
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+The first endpoint allows you to retrieve the rights for all the entities of the authenticated person.
+
+To do this, you must do a GET on `/ngsi-ld/v1/entityAccessControl/entities`.
+
+The following request parameters are supported: 
+
+    -  q: to restrict returned entities to the ones with a specific right. Only "rCanRead" and "rCanWrite" and "rCanAdmin" are accepted. A list is accepted (e.g, “q=rCanRead;rCanWrite”). This request parameter does not work when user are stellio admin. 
+
+    -  type: to restrict returned entities to a given entity type
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:19]:
```suggestion
    -  `type`: to restrict returned entities to a given entity type 
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+To do this, you must do a GET on `/ngsi-ld/v1/entityAccessControl/entities`.
+
+The following request parameters are supported: 
+
+    -  q: to restrict returned entities to the ones with a specific right. Only "rCanRead" and "rCanWrite" and "rCanAdmin" are accepted. A list is accepted (e.g, “q=rCanRead;rCanWrite”). This request parameter does not work when user are stellio admin. 
+
+    -  type: to restrict returned entities to a given entity type 
+
+    - options: with the sysAttrs value to get the system attributes.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:21]:
```suggestion
    - `options`: use `sysAttrs` value to get the system attributes.
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        ], 
+        “rCanAdmin”: [ 
+         … 
+        ], 
+        @context: […] 
+    }
+    ```
+    The body also contains the users who have right on it.  
+
+    3) If user use stellio without authentification the response sent the status 204 no content.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:81]:
```suggestion
    3) If user use Stellio without authentification, a 204 (No content) response code is returned.
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+        “name”: { 
+            “type”: “Property”, 
+            “value”: “egm” 
+        }, 
+        context: […] 
+    }
+    ```
+    The body contains id, type and name of the group. 
+
+    2) If user is not admin, all groups are returned:
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:107]: is admin instead?

File: docs/admin/upgrading_to_1.9.0.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+            “value”: “egm” 
+         }, 
+        “isMemberOf”: { 
+            “type”: “Property”, 
+            “value”: “true” 
+        }, 
+        @context: […] 
+    }
+    ```
+    The body contains id, type, name of the group and if user are member of this group.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:124]:
```suggestion
    The body contains id, type, name of the group and if user is member of this group. 
```

File: docs/admin/upgrading_to_1.9.0.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+        “isMemberOf”: { 
+            “type”: “Property”, 
+            “value”: “true” 
+        }, 
+        @context: […] 
+    }
+    ```
+    The body contains id, type, name of the group and if user are member of this group. 
+
+    3) If user use stellio without authentification the response sent the status 204 no content.
```
Comment (bobeal) [inline docs/admin/upgrading_to_1.9.0.md:126]:
```suggestion
    3) If user use Stellio without authentification, a 204 (No content) response code is returned.
```

**Comment by bobeal** [comment]
Other general notes:
- Use backticks for request parameters, ...
- Stellio, not stellio
- Avoid the use of "you"

File: docs/admin/upgrading_to_1.9.0.md (hunk 7 above)
Comment (gpoujol) [reply docs/admin/upgrading_to_1.9.0.md:107]: yes indeed

**Comment by bobeal** [comment]
I have rewritten part of the doc, I let you check and validate.

**Pull request merged**